File: The Core/1. Intro Gates/8. Phone Call/phoneCall.cs
int phoneCall(int min1, int min2_10, int min11, int s) {
    
    int centsLeft = s;
    int minutes = 0;
    
    if(min1 <= centsLeft){
        centsLeft -= min1;
        minutes += 1;
    } else {
        return minutes;
    }
    
    for(int i = 2; i <= 10; i++){
        if(min2_10 < centsLeft){
            centsLeft -= min2_10;
            minutes += 1;
        } else {
            return minutes;
        }
    }
    
    double rem = centsLeft/min11;
    minutes += Convert.ToInt32(Math.Floor(rem));
    
    return minutes;

}

File: The Core/Intro Gates/8. Phone Call/phoneCall.js
function phoneCall(min1, min2_10, min11, s) {
    var centsRemaining = s;
    var minutes = 0;
    
    if(min1 <= centsRemaining){
        centsRemaining -= min1;
        minutes += 1;
    } else {
        return minutes;
    }
    
    for(i = 2; i <= 10; i++){
        if(min2_10 < centsRemaining){
            centsRemaining -= min2_10;
            minutes += 1;
        } else {
            return minutes;
        }
    }
    
    var rem = centsRemaining/min11;
    minutes += parseInt(Math.floor(rem));
    
    return minutes;
}
